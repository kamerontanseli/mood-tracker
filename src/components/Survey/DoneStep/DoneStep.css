.DoneStep-title {
  text-align: center;
  margin: 50px 0;
}

.DoneStep-controls {
  text-align: center;
  margin-bottom: 50px;
}