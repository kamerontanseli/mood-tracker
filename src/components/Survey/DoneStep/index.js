import "./DoneStep.css";
import React from "react";
import { Link } from 'react-router-dom'
import Panel from "../../general/Panel";
import Header1 from "../../general/Header1";

const DoneStep = ({  }) => (
  <Panel>
    <div className="DoneStep">
      <div className="DoneStep-title">
        <Header1>Survey completed!</Header1>
      </div>
      <div className="DoneStep-controls">
        <Link to="/">Go to your insights</Link>
      </div>
    </div>
  </Panel>
);

export default DoneStep;
