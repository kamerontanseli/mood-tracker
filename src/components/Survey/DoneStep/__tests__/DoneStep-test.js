import React from "react";
import { shallow, mount } from "enzyme";
import renderer from "react-test-renderer";
import DoneStep from "..";

jest.mock("react-router-dom", () => ({
  Link: p => <a {...p} />
}));

describe("DoneStep", () => {
  it("should render", () => {
    const snapshot = renderer.create(<DoneStep />);
    expect(snapshot).toMatchSnapshot();
  });
});
