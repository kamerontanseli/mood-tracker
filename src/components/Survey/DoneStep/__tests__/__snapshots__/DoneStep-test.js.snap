// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DoneStep should render 1`] = `
<div
  className="Panel"
>
  <div
    className="DoneStep"
  >
    <div
      className="DoneStep-title"
    >
      <h1
        className="Header1"
      >
        Survey completed!
      </h1>
    </div>
    <div
      className="DoneStep-controls"
    >
      <a
        to="/"
      >
        Go to your insights
      </a>
    </div>
  </div>
</div>
`;
